### PR TITLE
Microsoft Graph Single User - Edit documentation

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-24630

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml [hunk 1]
```diff
@@ -111,6 +111,7 @@ configuration:
   required: true
   type: 0
   section: Collect
+  additionalinfo: Make sure you are logged in to this email during configuration and authentication, otherwise it will not work.
```
Comment (yucohen) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml:114]: Maybe something like:
 "As this is a single-user integration, please make sure to authenticate with the same user configured" ?

**Review by yucohen — APPROVED** [review]

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml (hunk 1 above)
Comment (yucohen) [reply Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml:114]: @ShirleyDenkberg WDYT?

Comment (ShirleyDenkberg) [reply Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml:114]: "As this is a single-user integration, make sure to authenticate with the same configured user."

**Comment by ShirleyDenkberg** [comment]
@michal-dagan @yucohen Doc review completed.

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml (hunk 1 above)
Comment (rshunim) [reply Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml:114]: @yucohen @ShirleyDenkberg Is the proposed sentence clear that the user should be connected to the email he enters in the configuration?

**Pull request merged**